src/plugins/image/image.yml

```diff
@@ -31,6 +34,12 @@ copyrightInfos:
 maximumWidth:
   en: Maximum width
   de: Maximale Breite
+clippedWidth:
+  en: Clipping actual width
+  de: Echte abgeschnittene Breite
```

Comment:
I think it would be more understandable to the user if we'd say: "Resulting (width|height)" / "Resultierende (Breite|Höhe)"

Reply:
I struggled for a good wording here, this is better, thanks!
Updated